docs: revised dark mode feature page (#26187)
electron/electron: docs/tutorial/dark-mode-guide.md

> # Supporting Dark Mode

Author: Unified the variations to `Dark Mode`

})
```

Next step is to update the `main.js` file to handle events from the Renderer
process. Depending on the received event (`dark-mode:toggle` or
`dark-mode:system`), the [`nativeTheme.themeSource`](../api/native-theme.md#nativethemethemesource)
API applies the theme on the system native elements (e.g. context menus on macOS).

```js
const { app, BrowserWindow, ipcMain, nativeTheme } = require('electron')

function createWindow () {
  const win = new BrowserWindow({
    width: 800,
    height: 600,
    webPreferences: {
      nodeIntegration: true
    }
  })

  win.loadFile('index.html')

  ipcMain.handle('dark-mode:toggle', () => {
    if (nativeTheme.shouldUseDarkColors) {
      nativeTheme.themeSource = 'light'
    } else {
      nativeTheme.themeSource = 'dark'
    }
    return nativeTheme.shouldUseDarkColors
  })

  ipcMain.handle('dark-mode:system', () => {
    nativeTheme.themeSouce = 'system'
  })
}

app.whenReady().then(createWindow)

app.on('window-all-closed', () => {
  if (process.platform !== 'darwin') {
    app.quit()
  }
})

app.on('activate', () => {
  if (BrowserWindow.getAllWindows().length === 0) {
    createWindow()
  }
})
```

> The final step is to add a bit of styling to enable dark mode for the web parts
> by leveraging the [`prefers-color-scheme`][prefer-color-scheme] CSS attribute.
> This part only follows the system theme.

Reviewer: > This part only follows the system theme.

Not necessarily true :P 

```suggestion
The final step is to add a bit of styling to enable dark mode for the web parts of
the UI by leveraging the [`prefers-color-scheme`][prefer-color-scheme] CSS attribute.
The value of `prefers-color-scheme` will follow your `nativeTheme.themeSource` setting.
```

> Create `styles.css` file and add the following lines:

Reviewer: ```suggestion
Create a `styles.css` file and add the following lines:
```

```css
@media (prefers-color-scheme: dark) {
  body { background:  #333; color: white; }
}

@media (prefers-color-scheme: light) {
  body { background:  #ddd; color: black; }
}
```

> After launching the Electron application, try to change modes by clicking the
> toggle button or reset the theme to system default:

Reviewer: ```suggestion
After launching the Electron application, try to change modes by clicking the
toggle button and then resetting the theme to system default:
```
